herd_behaviour.nlogo

```diff
@@ -1,72 +1,53 @@
-;; avoidance range radius
-;; following range radius
-;; percent_informed - proportion of informed individuals
-;; weight - value between 0 and 2 to weight there own direction preference
-;; number_herd_members - number of herd members
-;; aim-direction - d-dir in paper
+
 globals [
-  number_informed
-  number_uninformed
-  g-dir ;; preffered direction same for all informed turtles same
-  w ;; weight for all informed turtles same
+  ;; avoidance_range
+  ;; following_range
+  ;; percent_informed - proportion of informed individuals
+  ;; number_herd_members - number of herd members
+  ;; weigth - weight with which prefered direction is included
 ]
 
 turtles-own [
-  ;; want to maintain minimum distancs alpha
-  speed ;; individual speed
-  omega
+  speed ;; individual speed (modelled on agent level to add different speeds later)
   v-dir ;; direction vector
-  informed ;; boolean
-  ;; ci (position vector)
-  neighbours-avoid ;; neighours in alpha avoidance range
```

Comment:
as the formulas are calculated out of the neighbours direction, I think at each step the turtles have to know their avoid and follow neighbours